api/shared/state_types.go

```diff
@@ -9,6 +9,7 @@ type RawState []byte
 //
 // [1] https://github.com/nmstate/nmstate/blob/master/libnmstate/schemas/operational-state.yaml
 // +k8s:openapi-gen=true
```

Comment:
You dropped openapi targets in this PR. Does it make sense to keep these comments?

Reply:
Good catch, removing.